<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>IP-adress</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>201</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Port number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>201</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Command</string>
    </property>
   </widget>
   <widget class="QPushButton" name="executeButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>88</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="myIP">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="myPort">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="myCommand">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
